events/class_models/event_models.py

```diff
@@ -1,7 +1,61 @@
 from django.db import models
+from django.db.models import Count
 from django.core.exceptions import ValidationError
 
 
+class EventQuerySet(models.QuerySet):
+    def get_all_meetings(self):
+        """ get all the meetings from all the users """
+        return self.alias(participant_amount=Count('eventparticipant__event_id')).filter(participant_amount__gte=2)
+
+    def get_all_users_meetings(self, user_id):
```

Comment:
AFAIU this method get the all meeting of a single user , if soo i think it would be more readable to be user instead of users.

Reply:
done. thanks 😄